scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -258,6 +259,26 @@ object Execution {
     TransformedConfig(ex, c)
 
   /**
+   * Distribute the file across map/reduce nodes and
+   * give you instance of [[com.twitter.scalding.filecache.CachedFile]].
+   *
+   * The cached file available only inside execution.
+   * You can not return cached file as result of execution and must use inside closure.
+   *
+   * For multiple files you must nested your execution, see docs of [[com.twitter.scalding.filecache.DistributedCacheFile]]
```

Comment:
snippet available in DistributedCacheFile, and I put the link to it. do you want copy to here also?